### ISSUE TITLE
[osm] Footer links - accessibility

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-websites#1426]
The social media footer links are failing accessibility as they do not have discernible text

`<a class="icon-facebook" href="https://www.facebook.com/joomla" target="_blank"><span>Facebook</span></a>`

We can see from the code that the text is present but instead of being visually hidden it has **display:none** which removes the text completely from the dom.

```
ul.footer-social li a span {
    display: none;
}
```

If that css is removed from the template.css and the code is changed to use the **sr-only** class then it will still be visually hidden but will be accessible.

`<a class="icon-facebook" href="https://www.facebook.com/joomla" target="_blank"><span class="sr-only">Facebook</span></a>`

**Comment by conconnl** [comment]
I copied the sr-only CSS to the "ul.footer-social li a span" which should give the same result and doesn't require mod_menu override.

**Comment by brianteeman** [comment]
That worked - thanks
